### PR TITLE
Fix/tao 5747/response condition

### PR DESCRIPTION
**Pull request by zagovorichev**

### CODE REVIEW

**Review by krampstudio — CHANGES_REQUESTED** [review]
This change fix the problem for all PCIs but the "Graphing f(x)" PCI. If you try to add the "Fractions" PCI remove it and add again the "Graphing f(x)" PCI, there's still an error `uncaught exception: unexpected number of rc found`

Also the rule is to do a check on the code style when opening very old files, like moving variable declaration to the top of their scope or throwing `new Error` instead of  [a string](https://github.com/oat-sa/extension-parcc-tei/pull/3/files#diff-41c1a3d631f67aa9568fccc12ce91c24R59)
It would be great if you can make those changes too.

**Comment by zagovorichev** [comment]
@krampstudio Update

File: views/js/pciCreator/helper/responseCondition.js [hunk 1]
```diff
@@ -24,13 +24,15 @@ define(['lodash', 'jquery', 'taoQtiItem/qtiCreator/helper/xmlRenderer'], functio
...
+        var rcXml = $.parseXML(newResponseConditionXml);
+        var newRc = $rpXml[0].importNode(rcXml.documentElement, true);
         var responseIdentifier = interaction.attr('responseIdentifier');
         var responseDeclaration = interaction.getResponseDeclaration();
+        var $respVar = $(rcXml).find('variable[identifier="'+responseIdentifier+'"]');
 
         //prepare replacement criteria
         criteria = _.defaults(criteria || {}, {
-            responseIdentifierCount : 1
+            responseIdentifierCount : $respVar.length
```
Comment (zagovorichev) [inline views/js/pciCreator/helper/responseCondition.js:35]: Instead of this I could count all RESPONSE from template, of course if this has sense for you

**Review by krampstudio — APPROVED** [review]
It seems fixed now

**Pull request merged**